src/gleam/erlang/file.gleam

```diff
@@ -123,19 +123,32 @@ pub type Reason {
 pub external fn is_directory(path: String) -> Bool =
   "filelib" "is_dir"
 
-/// Returns true if path refers to a file, otherwise false.
+/// Returns true if path refers to a regular file, otherwise false.
```

Comment:
What does `regular` mean? Not a word I've come across before when talking about files. Would `is_file` be a clearer name?

Reply:
For e.g. the `test` command available in most shells

```sh
test
  -d FILE
      Returns true if FILE is a directory.
  -e FILE
      Returns true if FILE exists.
  -f FILE
      Returns true if FILE is a regular file.
```

The main issue here is Erlang's choice of names for these functions.
`filelib:is_file` is `test -e`
`filelib:is_regular` is `test -f`

I'm open to other names, but I think it shouldn't be confusing. Currently, I'd say it's confusing.

`file.is_regular` seems fine to me. `file.is_file` would potentially be confusing again if e.g. someone overlooks `file.exists`.

At first I went with `is_regular_file`, but then thought it was redundant given the module name.

Reply:
More specifically, as far as I know, `regular` means it's not a directory, link, or any other "special" type of file.

Reply:
That's interesting. I'm still not sure what it means, could we add the definition of `regular` to the docs?

Reply:
According to Elixir docs and a bit of testing, a "regular" file is one that isn't a "directory, socket, symbolic link, named pipe, or device file."